src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -157,45 +157,42 @@ public class SendReceiveTest : ClientWebSocketTestBase
                 var recvBuffer = new byte[100];
                 var recvSegment = new ArraySegment<byte>(recvBuffer);
 
-                try
+                for (int i = 0; i < tasks.Length; i++)
                 {
-                    for (int i = 0; i < tasks.Length; i++)
-                    {
-                        tasks[i] = cws.ReceiveAsync(recvSegment, cts.Token);
-                    }
+                    tasks[i] = cws.ReceiveAsync(recvSegment, cts.Token);
+                }
 
-                    Task.WaitAll(tasks);
+                Exception e = Record.Exception(() => Task.WaitAll(tasks));
```

Comment:
Because it's supposed to be more along the lines of [AAA](https://msdn.microsoft.com/en-us/library/hh694602.aspx#Anchor_3), which ends up separating the throw/catch from examining the value (here, the exception) itself.  
I only actually did this because I'm not the first to put this in, although I wasn't particularly planning on general replacement of the existing asserts otherwise (absent a more specific directive in one way or the other).  The null check could be potentially removed, as `IsType` should catch that as well...